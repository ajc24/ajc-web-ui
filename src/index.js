/* Export all of the header components */
export { default as Header } from './ui-elements/header/Header';

/* Export all of the icon components */
export { default as SmallDownArrowIcon } from './ui-elements/icons/SmallDownArrowIcon';
export { default as SmallRightArrowIcon } from './ui-elements/icons/SmallRightArrowIcon';
