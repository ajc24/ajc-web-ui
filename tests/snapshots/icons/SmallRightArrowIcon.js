import 'jsdom-global/register';
import React from 'react';
import { TestDev } from 'ajc-jest-enzyme';
import { SmallRightArrowIcon } from '../../../src';

describe('SmallRightArrowIcon', () => {
  describe('Default props and rendering - Component in black colour', () => {
    let jsonSnapshot;

    beforeAll(() => {
      jsonSnapshot = TestDev.createSnapshot(
        <React.Fragment>
          <SmallRightArrowIcon />
        </React.Fragment>
      );
    });
    
    it('verifies the snapshot for the component', () => {
      expect(jsonSnapshot).toMatchSnapshot();
    });
  });

  describe('Transferred props and rendering - Component in white colour', () => {
    let jsonSnapshot;

    beforeAll(() => {
      jsonSnapshot = TestDev.createSnapshot(
        <React.Fragment>
          <SmallRightArrowIcon colour="white" />
        </React.Fragment>
      );
    });

    it('verifies the snapshot for the component', () => {
      expect(jsonSnapshot).toMatchSnapshot();
    });
  });
});
