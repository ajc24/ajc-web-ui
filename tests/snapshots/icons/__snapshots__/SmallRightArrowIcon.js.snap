// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallRightArrowIcon Default props and rendering - Component in black colour verifies the snapshot for the component 1`] = `
<div
  className="ajc-flex-row ajc-justify-left"
  role="presentation"
>
  <div
    className="ajc-flex-row ajc-small-right-arrow-icon ajc-border-colour-black"
  />
</div>
`;

exports[`SmallRightArrowIcon Transferred props and rendering - Component in white colour verifies the snapshot for the component 1`] = `
<div
  className="ajc-flex-row ajc-justify-left"
  role="presentation"
>
  <div
    className="ajc-flex-row ajc-small-right-arrow-icon ajc-border-colour-white"
  />
</div>
`;
